pallets/creator-staking/src/functions.rs

```diff
@@ -163,70 +163,70 @@ impl<T: Config> Pallet<T> {
 
         stake_info.total = stake_info.total.saturating_sub(amount_to_unstake);
 
-        staker_info
+        backer_info
             .unstake(current_era, amount_to_unstake)
             .map_err(|_| Error::<T>::CannotChangeStakeInPastEra)?;
 
-        Self::ensure_max_era_stake_items_not_exceeded(staker_info)?;
+        Self::ensure_max_era_stake_items_not_exceeded(backer_info)?;
 
         Ok(amount_to_unstake)
     }
 
-    /// Update the ledger for a staker. This will also update the stash lock.
+    /// Update the locks for a backer. This will also update the stash lock.
     /// This lock will lock the entire funds except paying for further transactions.
-    pub(super) fn update_ledger(staker: &T::AccountId, ledger: BackerLocksOf<T>) {
-        if ledger.is_empty() {
-            Ledger::<T>::remove(staker);
-            T::Currency::remove_lock(STAKING_ID, staker);
+    pub(super) fn update_backer_locks(backer: &T::AccountId, backer_locks: BackerLocksOf<T>) {
+        if backer_locks.is_empty() {
+            BackerLocksByAccount::<T>::remove(backer);
+            T::Currency::remove_lock(STAKING_ID, backer);
         } else {
-            T::Currency::set_lock(STAKING_ID, staker, ledger.locked, WithdrawReasons::all());
-            Ledger::<T>::insert(staker, ledger);
+            T::Currency::set_lock(STAKING_ID, backer, backer_locks.total_locked, WithdrawReasons::all());
+            BackerLocksByAccount::<T>::insert(backer, backer_locks);
         }
     }
 
-    /// Update the staker info for the `(staker, creator_id)` pairing.
-    /// If staker_info is empty, remove it from the DB. Otherwise, store it.
-    pub(super) fn update_staker_info(
-        staker: &T::AccountId,
-        creator_id: SpaceId,
-        staker_info: StakesInfoOf<T>,
+    /// Update the backer info for the `(backer, creator_id)` pairing.
+    /// If backer_info is empty, remove it from the DB. Otherwise, store it.
+    pub(super) fn update_backer_info(
+        backer: &T::AccountId,
+        creator_id: CreatorId,
+        backer_info: StakesInfoOf<T>,
```

Comment:
backer vs Stakes - on one line. Looks a bit strange.
Should `StakesInfoOf` be renamed to `BackerInfoOf`

Reply:
No. `StakesInfo` structure holds specifically information about backer stakes.
To align with the context of this function, it may be:

```suggestion
        backer_stakes: StakesInfoOf<T>,
```